pkg/services/accesscontrol/evaluator/evaluator.go

```diff
@@ -2,9 +2,11 @@ package evaluator
 
 import (
 	"context"
+	"errors"
+	"fmt"
+	"strings"
 
-	"github.com/gobwas/glob"
-
+	"github.com/grafana/grafana/pkg/cmd/grafana-cli/logger"
```

Comment:
Replace this with

```suggestion
	"github.com/grafana/grafana/pkg/infra/log"
```
https://pkg.go.dev/github.com/grafana/grafana/pkg/infra/log#New

and then create a logger for the package

```go
var logger = log.New("accesscontrol.evaluator")
```
or
```go
var logger = log.New("accesscontrol")
```

The logger name serves two functions, partly it makes makes it easier to sift through the logs and partly it allows you to set up different log levels for different features in grafana.ini, mostly useful to have a particular feature to print debug level messages.

I usually don't think it's necessary to have multiple loggers for a single service, but that's just a matter of preference.

Reply:
Thanks, I always manage to include the wrong one